package.json

```diff
@@ -7,7 +7,9 @@
     "server": "nodemon server/index.js",
     "app": "webpack -d --watch",
     "seed": "node data/addData.js",
-    "test": "jest"
+    "test": "jest",
+    "psqlSeed": "psql listinginfo < psql.sql",
+    "psqlShell": "psql listinginfo"
```

Comment:
I love adding scripts to package.json.
I would recommend adding these scripts w/ instructions to your repo's README.md file.
Makes repo re-initialization / sharing clean and easy.